pkg/convert/tf.go

```diff
@@ -2657,6 +2699,54 @@ See https://www.pulumi.com/docs/iac/concepts/options/deletebeforereplace/ for de
 	}
 }
 
+func convertSandboxModuleCall(
+	state *convertState,
+	scopes *scopes,
+	sandboxedModule *SandboxModule,
+) (hclwrite.Tokens, *hclwrite.Block, hclwrite.Tokens) {
+	// We translate module calls into components
+	moduleCall := sandboxedModule.moduleCall
+	path := "module." + moduleCall.Name
+	pulumiName := scopes.roots[path].Name
+
+	token := sandboxedModule.packageName + ":index:Module"
+	labels := []string{pulumiName, token}
+	block := hclwrite.NewBlock("resource", labels)
+	blockBody := block.Body()
+
+	// Does this resource have a count? If so set the "range" attribute
```

Comment:
Interesting is there a test that demonstrates this? You can call a module with a count?

Reply:
>  You can call a module with a count?

I believe you can, both with `count` and `for_each` to instantiate multiple instances of the thing. I can add an example to the test